tests/test_train.py

```diff
@@ -29,33 +29,29 @@ def test_train(cfg, tmpdir):
     cfg_tmp.eval.predict_vids_after_training = True
     cfg_tmp.eval.save_vids_after_training = True
 
-    # change directory to save outputs elsewhere
-    os.chdir(tmpdir)
-
-    # train model
-    train(cfg_tmp)
-
-    # change directory back
-    os.chdir(pwd)
+    # temporarily change working directory to temp output directory
+    with tmp_path:
+        # train model
+        train(cfg_tmp)
 
     # ensure labeled data was properly processed
-    assert os.path.isfile(os.path.join(tmpdir, "config.yaml"))
-    assert os.path.isfile(os.path.join(tmpdir, "predictions.csv"))
-    assert os.path.isfile(os.path.join(tmpdir, "predictions_pca_multiview_error.csv"))
-    assert os.path.isfile(os.path.join(tmpdir, "predictions_pca_singleview_error.csv"))
-    assert os.path.isfile(os.path.join(tmpdir, "predictions_pixel_error.csv"))
+    assert (tmp_path / "config.yaml").is_file
```

Comment:
this is nice. does this `(tmp_path / "config.yaml")` construction work cross-platform? (i.e. windows)

Reply:
Yes